builtin/logical/ssh/path_lookup.go

```diff
@@ -0,0 +1,100 @@
+package ssh
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathLookup(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "lookup",
+		Fields: map[string]*framework.FieldSchema{
+			"ip": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "IP address of target",
+			},
+		},
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.WriteOperation: b.pathLookupWrite,
+		},
+		HelpSynopsis:    pathLookupSyn,
+		HelpDescription: pathLookupDesc,
+	}
+}
+
+func (b *backend) pathLookupWrite(req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	ipAddr := d.Get("ip").(string)
+	if ipAddr == "" {
+		return logical.ErrorResponse("Missing 'ip'"), nil
+	}
+	ip := net.ParseIP(ipAddr)
+	if ip == nil {
+		return logical.ErrorResponse(fmt.Sprintf("Invalid IP '%s'", ip.String())), nil
+	}
+
+	keys, err := req.Storage.List("policy/")
+	if err != nil {
+		return nil, err
+	}
+	if len(keys) == 0 {
```

Comment:
It's easier to just initialize the other variable as an empty slice so that they get the same output but with only one code path to maintain